File: Backend/src/Repos/Models/XML_Model/XML4.js
const mongoose = require('mongoose');

const XML4Schema = new mongoose.Schema({
  id:             { type: Number, required: false },
  Ma_Lk:          { type: String, default: null },
  Stt:            { type: Number, default: null },
  Ma_Dich_Vu:     { type: String, default: null },
  Ma_Chi_So:      { type: String, default: null },
  Ten_Chi_So:     { type: String, default: null },
  Gia_Tri:        { type: String, default: null },
  Don_Vi_Do:      { type: String, default: null },
  Mo_Ta:          { type: String, default: null },
  Ket_Luan:       { type: String, default: null },
  Ngay_Kq:        { type: String, default: null },
  Ma_Bs_Doc_Kq:   { type: String, default: null },
  Du_Phong:       { type: String, default: null }
}, {
  timestamps: true,
  strict: false
});

module.exports = mongoose.model('XML4', XML4Schema);

File: WPF_GiamDinhBaoHiemYTe/Repos/Model/Patietn_XML_data/XML4.cs
using CommunityToolkit.Mvvm.ComponentModel;
using WPF_GiamDinhBaoHiem.Services.Interface;

namespace WPF_GiamDinhBaoHiem.Repos.Model
{
    public partial class XML4 : ObservableObject, IHasStt
    {
        [ObservableProperty] private int id;
        [ObservableProperty] private string? ma_Lk;
        [ObservableProperty] private int? stt;
        [ObservableProperty] private string? ma_Dich_Vu;
        [ObservableProperty] private string? ma_Chi_So;
        [ObservableProperty] private string? ten_Chi_So;
        [ObservableProperty] private string? gia_Tri;
        [ObservableProperty] private string? don_Vi_Do;
        [ObservableProperty] private string? mo_Ta;
        [ObservableProperty] private string? ket_Luan;
        [ObservableProperty] private string? ngay_Kq;
        [ObservableProperty] private string? ma_Bs_Doc_Kq;
        [ObservableProperty] private string? du_Phong;
        [ObservableProperty] private bool isError;
    }
}
